08-transport.Rmd

```diff
@@ -102,12 +109,13 @@ View(cw0103)
 Despite impressive walking and cycling statistics, the city has a major congestion problem.
 Part of the solution is to continue increase the proportion of trips made by cycling.
 Cycling has a greater potential to replace car trips than walking because of the speed of this mode, around 3-4 times faster than walking (with typical [speeds](https://en.wikipedia.org/wiki/Bicycle_performance) of 15-20 km/h vs 4-6 km/h for walking).
-There is therefore a [plan](http://www.cyclingweekly.com/news/interview-bristols-mayor-george-ferguson-24114) to double the mode share of cycling by 2020.
+There is an ambitious [plan](http://www.cyclingweekly.com/news/interview-bristols-mayor-george-ferguson-24114) to double the share of cycling by 2020.
 
-In this policy context the aim of this chapter, beyond demonstrating how geocomputation with R can be used to support sustainable transport planning, is to provide evidence for decision-makers in Bristol to decide how best to increase the mode share of walking and cycling in particular in the city.
+In this policy context the aim of this chapter, beyond demonstrating how geocomputation with R can be used to support sustainable transport planning, is to provide evidence for decision-makers in Bristol to decide how best to increase the share of walking and cycling in particular in the city.
 This high-level aim will be met via the following objectives:
 
 - Describe the geographical pattern of transport behavior in the city.
+<!-- potential bottlenecks? you mean where cycling/walking cannot be further increased? --> 
 - Identify key nodes and potential bottlenecks where walking and cycling potential is likely to be highest.
```

Comment:
Potential bottlenecks? You mean where cycling/walking cannot be further increased?